test/testapps/TestContentPackage/build/TestContentPackage.props

```diff
@@ -0,0 +1,19 @@
+﻿<Project>
+  <PropertyGroup>
+    <!-- Update this to match your package ID exactly -->
+    <_PackageId>TestContentPackage</_PackageId>
+    <_ContentDir>$(MSBuildThisFileDirectory)..\content\</_ContentDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <!-- All files under "content" will be included with the Blazor app build output -->
+    <BlazorPackageContentFile Include="$(_ContentDir)**" SourcePackage="$(_PackageId)" />
```

Comment:
For this I would use content and an Additional metadata attribute inside the content. That way you can piggyback on all the features of content.
The only reason I didn't do this for asmjs, wasm and other static content in the blazor core outputs is because it doesn't surface to the user.

That said, this is just a suggestion, you can give content a try or just leave it like this for now.

Another completely different option can be to just define stuff in the package as content and make it go into
(wwwroot|dist)/_blazorcontent/(js|css)/** in the consuming project and have blazor include anything under _blazorcontent/(js|css)/**.* in the index.html

That way as a package author you just have to care about putting stuff in the right folder and everything else happens automatically.

Reply:
> Another completely different option can be to just define stuff in the package as content and make it go into

@javiercn I tried to do this at some length, by putting `<PackageCopyToOutput>true</PackageCopyToOutput>` on the `<Content>` items in the source package and using `PackagePath` to make them land in the correct place in the output directory. It works correctly at build time.

This *would* a be better way to do it, except for one severe problem, which is that all `<Content>` items appear in the *consuming* project's Solution Explorer tree as files (with little "link" icons). This makes the consuming project look messy, or at least adds totally unfamiliar items to the tree.

If you know of a way of using `<Content>` and *not* having the items show up in the consuming project, please let me know! I'm guessing it's not currently possible due to https://github.com/NuGet/Home/issues/4856 still being unresolved.

Reply:
I do believe that you can use the <Visible>false</Visible> node in MSBuild to make them not show up in VS https://msdn.microsoft.com/en-us/library/bb629388.aspx

So my idea is that if you put the content on the right folder path then you can have msbuild in our targets that based on the path updates the metadata for the itemgroup. (And this can be just at the top level so it applies globally)

Reply:
Are you talking about `Visible="false"`? I did try that, and although it hid the files, it did not hide the directories. It looked very weird. If you're able to make this work and hide both please let me know!

Reply:
@SteveSandersonMS Yes. @rynowak This is also what you mentioned to me before, isn't it? To just use Content and use ```<Visible>false</Visible>``` do you know about the folders showing up?